egui_glow/src/painter.rs

```diff
@@ -481,6 +490,7 @@ impl Painter {
 
                 let data: &[u8] = bytemuck::cast_slice(image.pixels.as_ref());
 
+                self.set_texture_filter(delta.filter.into());
```

Comment:
I would prefer the stateful `set_texture_filter` was removed and that the filter was passed by argument to `upload_texture_srgb` - but I can also do that at a later time.

Reply:
Good point! I'll update that real quick.